docs/modules/ROOT/pages/custom-config.adoc

```diff
@@ -154,4 +86,4 @@ The Hazelcast Platform Operator communicates with the Hazelcast cluster, requiri
 
 === Security configuration
 
-When configuring `security` with enabled `client-authentication`, you must provide a user with `all` permissions and a simple authentication method. The client and the operator will use this user to connect to and manage the cluster.
+The security configuration is not accessible through custom configuration. To configure security use `Security` section of the Hazelcast CR.
```

Comment:
Could you put link for operator security doc?

Reply:
I think we should not put security document directly that it includes a couple of other unrelated thing like TLS etc. which makes it confusing. Maybe after adding all features, we can add a sentence in here like "to configure client simple and jaas auth, and member auth you can use the security section in the Hazelcast CR" by putting reference for simple and jaas auth and member auth.

Reply:
I mean this page --> docs/modules/ROOT/pages/client-simple-auth.adoc

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/260/commits/60945796be9cb7a3116a5875383423c4b40d5048